src/serde.rs

```diff
@@ -71,6 +71,42 @@ where
     }
 }
 
+/// Byte slice wrapper to serialize as a hex string in lowercase characters.
+pub struct SerializeBytesAsHex<'a>(pub(crate) &'a [u8]);
```

Comment:
Have you actually tested the API? I now see there's no way to create the struct. :)

I think we can simply make the field `pub`.

Reply:
Holy moly, what a crap PR. Sorry to waste your time.

Reply:
There are no invariants so I made the inner field `pub`.